sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -90,6 +90,8 @@
           <zeebe:header key="appealsStudentFinancialInformationAppealData" value="$.appeals.studentFinancialInformationAppeal.submittedData" />
           <zeebe:header key="appealsStudentAdditionalTransportationAppealData" value="$.appeals.studentAdditionalTransportationAppeal.submittedData" />
           <zeebe:header key="appealsPartnerInformationAndIncomeAppealData" value="$.appeals.partnerInformationAndIncomeAppeal.submittedData" />
+          <zeebe:header key="appealsStudentDependantsAppealData" value="$.appeals.studentDependantsAppeal.submittedData.dependants[*][&#39;dateOfBirth&#39;,&#39;declaredOnTaxes&#39;,&#39;attendingPostSecondarySchool&#39;]" />
+          <zeebe:header key="appealsStudentHasDependantsAppealData" value="$.appeals.studentDependantsAppeal.submittedData.hasDependents" />
```

Comment:
As per our discussion `appealsStudentHasDependantsAppealData` should use e. `appealsStudentHasDependentsAppealData`